.github/workflows/non-omv.yml

```diff
@@ -49,17 +53,19 @@ jobs:
         cd SBML
         ./validateAll.sh
 
-    - name: Test test_suite output regeneration
+    - name: test omex file creation
       run: |
-        cd SBML
-        pip install pymetadata docker "requests<2.32.0"
-        python create_minimal_omex_archive.py
```

Comment:
It might be good to still test the create_minimal_omex_archive.py script, to make sure it still works, or consolidate it with the other biosim docker script.

Reply:
it's already largely redundant due to test_biosimulators_docker.py